test/unit/spec/v2/models/AppState_spec.js

```diff
@@ -25,21 +25,17 @@ describe('v2/models/AppState', function() {
       this.initAppState({ idx: { actions: { cancel: 'invalid' } } });
       expect(this.appState.shouldShowSignOutLinkInCurrentForm()).toBe(false);
     });
-    it('returns false if the currentFormName is in FORMS_WITHOUT_SIGNOUT', () => {
-      this.initAppState({ idx: { actions: { cancel: () => {} } } }, FORMS_WITHOUT_SIGNOUT[0]);
```

Comment:
we still need a test for `FORMS_WITHOUT_SIGNOUT`, can we add it back? The test for `FORMS_FOR_VERIFICATION` is that one that is no longer needed

Reply:
Woah - good catch. I think I got too aggressive with my code deletion and missed this when reviewing. Thank you!